autocue2.liq

```diff
@@ -5,416 +5,389 @@
 
 # --- Copy-paste Azuracast LS Config, second input box BEGIN ---
 
-# Initialize settings for autocue2 protocol
-let settings.autocue2 = ()
+# Initialize settings for cue_file autocue implementation
+let settings.autocue.cue_file = ()
 
-let settings.autocue2.path =
+let settings.autocue.cue_file.path =
   settings.make(
     description=
-      "Path of the autocue2 binary.",
+      "Path of the cue_file binary.",
     "cue_file"
   )
 
-let settings.autocue2.timeout =
+let settings.autocue.cue_file.fade_in =
   settings.make(
     description=
-      "Timeout (in seconds) for autocue2 executions.",
+      "Default fade-in duration if no specified by the user.",
+    0.1
+  )
+
+let settings.autocue.cue_file.fade_out =
+  settings.make(
+    description=
+      "Default fade-out duration if no specified by the user.",
+    3.
+  )
+
+let settings.autocue.cue_file.metadata_override =
```

Comment:
I moved disabling processing based on previous metadata down to the implementation so that this one can accept pre-existing fade duration while the internal implementation skips in similar cases.

Reply:
Do we need this as a setting? It feels more logical to just add these as params to `crossfade`/`cross`/`fade_out`. Will those still work? Because one might want to have _different_ fades, like in a "live_aware_crossfade" with an `add` that has fade values set.

Reply:
Feel free to implement whatever logic you want!